modules/convert/repository.go

```diff
@@ -106,6 +107,20 @@ func innerToRepo(repo *repo_model.Repository, mode perm.AccessMode, isParent boo
 		}
 	}
 
+	var transfer *api.RepoTransfer
+	if repo.Status == repo_model.RepositoryPendingTransfer {
+		t, err := models.GetPendingRepositoryTransfer(repo)
+		if err != nil && !models.IsErrNoPendingTransfer(err) {
+			log.Debug("GetPendingRepositoryTransfer", err)
+		} else {
+			if err := t.LoadAttributes(); err != nil {
+				log.Debug("LoadAttributes of RepoTransfer", err)
```

Comment:
Here and above, I think we can log these unexpected errors as WARN or ERROR, it makes debug easier.

Reply:
https://github.com/go-gitea/gitea/pull/17963/commits/70841452d48b0b3eebc6aca8647e7e7e8baa74a2